server/src/com/thoughtworks/go/config/GoConfigFileDao.java

```diff
@@ -177,36 +141,6 @@ public String md5OfConfigFile() {
         return cachedConfigService.currentConfig().getMd5();
     }
 
-    public void updatePipelineGroup(final PipelineConfigs pipelineConfigs, final String groupName, final String md5) {
```

Comment:
You can edit the pipeline group through the ui when the admin sets pipeline group permissions. Are you sure this API is not called?